maven/bnd-maven-plugin/README.md

```diff
@@ -85,6 +85,7 @@ For further usage information, see the integration test projects under the inclu
 |`bnd`                  | Bnd instructions for this project specified directly in the pom file. This is generally be done using a {@code <![CDATA[]]>} section. If the projects has a `bndfile` configuration property or a file in the default location `bnd.bnd`, then this configuration element is ignored. |
 |`manifestPath`         | Specify the path to a manifest file to use. _Defaults to `${project.build.outputDirectory}/META-INF/MANIFEST.MF`._|
 |`skip`                 | Skip the project. _Defaults to `false`._ Override with property `bnd.skip`.|
+|`includeAllOutputs` | Include all build outputs for analysis by **bnd**. *Defaults to `true`*. If set to `false`, the build outputs are added to the analyzer's classpath instead, allowing for selective analysis. *Actually excluding contents from the jar is done using `maven-jar-plugin`'s `excludes/exclude` configuration. See [here](https://maven.apache.org/plugins/maven-jar-plugin/examples/include-exclude.html) for details.* |
```

Comment:
I think this is a terrible option name. In the context of the bnd-maven-plugin, the outputs are the bundle. It it really 'include standard inputs'. Or 'include complete target/classes folder'.